Meadowlark-js/backends/meadowlark-mongodb-backend/docker/.gitignore

```diff
@@ -0,0 +1,2 @@
+file.key
+debezium-mongodb.json
```

Comment:
The debezium connector configuration file, copied from an example with username/password added